client/src/App.js

```diff
@@ -4,14 +4,16 @@ import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
+import List from "./pages/List";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route path="/list" component={List} />
+        {/* <Route path="/" component={LandingPage} /> */}
```

Comment:
You'll want to uncomment this line because it could interfere with Carl's PR when merging

Reply:
Thank you, I now have it back in